### PR TITLE
[RM Ch6]: Linewrap to 120 chars

### PR DESCRIPTION
**Pull request by CsatariGergely**
Partially closes #2834

### CODE REVIEW

**Comment by CsatariGergely** [comment]
I would need some help @gkunz @karinesevilla @collivier . I have some cells where the content of the cell is a single string and contains more character than the cell. 

image

The result will be this: 

image

How can I indicate in rst that my string is still going on even when my cell is not big enough? :)

File: doc/ref_model/chapters/chapter06.rst [hunk 1]
```diff
@@ -241,9 +517,24 @@ Intra-Cloud Infrastructure Interfaces
...
 ---------------------------
 
+<<<<<<< HEAD
+An operational cloud needs a set of standard services to function. Services such as NTP for time synchronization, DHCP
+for IP address allocation, DNS for obtaining IP addresses for domain names, and LBaaS (version 2) to distribute incoming
+SSrequests amongst a pool of designated resources.
+
+=======
 An operational cloud needs a set of standard services to function. Services such as NTP for time synchronization, DHCP for IP address allocation, DNS for obtaining IP addresses for domain names, and LBaaS (version 2) to distribute incoming requests amongst a pool of designated resources.
+>>>>>>> master
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter06.rst:540]: something went wrong here

File: doc/ref_model/chapters/chapter06.rst [hunk 2]
```diff
@@ -82,21 +181,197 @@ Tenant          +      +           +      +      Create/delete only by Cloud Zon
 Virtual compute +      +           +      +      Create/delete by VDC users with appropriate role. Additional operations would include suspend/unsuspend
 Virtual network +      +    +      +      +      Create/delete by VDC users with appropriate role
 =============== ====== ==== ====== ====== ====== ===========================================================================================================
+>>>>>>> master
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter06.rst:184]: remove

File: doc/ref_model/chapters/chapter06.rst [hunk 3]
```diff
@@ -60,6 +107,58 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
...
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Tenant       | +    | +  |      | +    | +    | Create/delete only by Cloud Zone administrators                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Virtual      | +    | +  |      | +    | +    | Create/delete by VDC users with appropriate role. Additional         |
+| compute      |      |    |      |      |      | operations would include suspend/unsuspend                           |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Virtual      | +    | +  | +    | +    | +    | Create/delete by VDC users with appropriate role                     | 
+| network      |      |    |      |      |      |                                                                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+=======
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter06.rst:161]: remove

File: doc/ref_model/chapters/chapter06.rst [hunk 4]
```diff
@@ -60,6 +107,58 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
 Tenant Level APIs
 ~~~~~~~~~~~~~~~~~
 
+<<<<<<< HEAD
```
Comment (gkunz) [inline doc/ref_model/chapters/chapter06.rst:110]: remove

**Review by gkunz — CHANGES_REQUESTED** [review]

File: doc/ref_model/chapters/chapter06.rst (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst:540]: Uh-oh, that is an unhandled merge conflict.

File: doc/ref_model/chapters/chapter06.rst (hunk 4 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst:110]: Again a half done merging...

File: doc/ref_model/chapters/chapter06.rst (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst]: Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)

File: doc/ref_model/chapters/chapter06.rst (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst]: Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)

File: doc/ref_model/chapters/chapter06.rst (hunk 3 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst]: Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)

File: doc/ref_model/chapters/chapter06.rst (hunk 4 above)
Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst]: Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)

File: doc/ref_model/chapters/chapter06.rst [hunk 5]
```diff
@@ -60,178 +107,223 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
...
...d in     |                       |                        |       +--------+---------------+------------------------------------+ RegisterForAccEventRe | RegisterForAccEventRes | VNF → | Input  | vnfEventHandl | the handler for NFVI to use when   | quest                 | ponse                  | NFVI  |        | erId          | notifying the VNF of the event     |-----------------------+------------------------+-------+--------+---------------+------------------------------------+                       |                        | NFVI  | Input  | vnfEventHandl | Handler used by VNF registering    |                       |                        | → VNF |        | erId          | for this event                     | AccEventNotificationR | AccEventNotificationRe |       +--------+---------------+------------------------------------+ equest                | sponse                 |       | Input  | accEventMetaD |                                    |                       |                        |       |        | ata           |                                    |-----------------------+------------------------+-------+--------+---------------+------------------------------------+
```
Comment (ulikleber) [inline doc/ref_model/chapters/chapter06.rst:188]: Something screwed up here....

Comment (CsatariGergely) [reply doc/ref_model/chapters/chapter06.rst]: Good catch. Fixed in [1112060](https://github.com/cntt-n/CNTT/pull/2931/commits/11120609e6cda9a655d10194dcbacb9987210281)

**Review by gkunz — APPROVED** [review]

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@ulikleber, can you re-review and approve if possible?  Thanks.

**Review by ulikleber — APPROVED** [review]

**Pull request merged**